operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -400,20 +418,31 @@ Please refer [iSCSI Metrics](https://github.com/Cray-HPE/sbps-marshal/blob/main/
 
 ## Steps to switch from DVS to iSCSI
 
+This involves this sequence of operations:
+
 * Disable DVS
 * Uninstall CPS
 * Enable iSCSI SBPS
```

Comment:
Question: SBPS is default and already setup right?
I am somewhat confused since title says `Steps to switch from DVS to iSCSI`, but that should be done via installation.  If this is talking about removing CPS/DVS, then maybe it would be nice to say it here and also give the titles for the related docs like: Uninstall_CPS.md  and maybe docs that shows how to stop loading DVS on compute node or removing it?

Reply:
This we thought of documenting for the scenario where user wants to permanently switch to iSCSI.Thinking on better heading / text here.

Reply:
@kazuya-hpe Going to change this title as 'Steps to disable DVS" Hope it is fine.

Reply:
I changed the ordering of the topics as well.